base/applications/rapps/unattended.cpp

```diff
@@ -350,15 +350,18 @@ ParseCmdAndExecute(LPWSTR lpCmdLine, BOOL bIsFirstLaunch, int nCmdShow)
     {
         // Check whether the RAPPS MainWindow is already launched in another process
         HANDLE hMutex;
+        WCHAR szWindowText[200];
+        LPCWSTR pszMutex = bAppwizMode ? L"RAPPSAPPWIZ" : szWindowClass;
+        LoadStringW(hInst, bAppwizMode ? IDS_APPWIZ_TITLE : IDS_APPTITLE, szWindowText, _countof(szWindowText));
```

Comment:
It may be of interest to change this by:
```c++
const CStringW szWindowText(MAKEINTRESOURCEW(bAppwizMode ? IDS_APPWIZ_TITLE : IDS_APPTITLE));
```
instead.

Reply:
CStringW would probably use the heap? This object might stay alive for the duration of the program since it's in the same scope as the main window loop. That is why I used this buffer.

Reply:
In any case it uses memory (now it's the stack, so for the duration of the program you've now less stack available).
Or you could use local scope trick, like this:
```c++
// other variables
{
CStringW szWindowText(...);
do_stuff(szWindowText);
}
// continue as usual
```
Not that elegant but it's done elsewhere.

Reply:
The compiler might be smart enough to place CMainWindow in the same stack space.

I don't like the scope hack, it needs to live too long for that.

I could set the string to empty when I don't need it anymore?

Reply:
> I could set the string to empty when I don't need it anymore?

Could work as well (`szWindowText.Empty();`)